recipe/meta.yaml

```diff
@@ -1,59 +1,56 @@
 {% set name = "gdk-pixbuf" %}
-{% set version = "2.42.6" %}
-{% set sha256 = "c4a6b75b7ed8f58ca48da830b9fa00ed96d668d3ab4b1f723dcf902f78bde77f" %}
+{% set version = "2.42.8" %}
+{% set sha256 = "84acea3acb2411b29134b32015a5b1aaa62844b19c4b1ef8b8971c6b0759f4c6" %}
 
 package:
     name: {{ name|lower }}
     version: {{ version }}
 
 source:
     sha256: {{ sha256 }}
-    url: http://ftp.gnome.org/pub/gnome/sources/{{ name }}/{{ '.'.join(version.split('.')[:2]) }}/{{ name }}-{{ version }}.tar.xz
+    url: https://download.gnome.org/sources/{{ name }}/{{ '.'.join(version.split('.')[:2]) }}/{{ name }}-{{ version }}.tar.xz
     patches:
       - 0001-changed-perl-script-to-use-env.patch
-      #- meson-rpaths-common.patch
-      #- meson-rpaths-non-macos.patch  # [not osx]
 
 build:
-  number: 5
-  skip: true  # [win and vc<14]
+  number: 0
   run_exports:
     # https://abi-laboratory.pro/?view=timeline&l=gdk-pixbuf
     - {{ pin_subpackage('gdk-pixbuf', max_pin='x') }}
   ignore_run_exports:
     - zlib
-    - jpeg
+    - jpeg    # [win]
+    - libffi  # [win]
 
 requirements:
   build:
     - pkg-config
     - {{ compiler('c') }}
-    # See https://github.com/conda-forge/atk-feedstock/pull/14#issuecomment-610061908
     - meson ==0.56.2  # [osx and arm64]
```

Comment:
The only version of `meson` that seems to be specified is `0.55.3`

https://gitlab.gnome.org/GNOME/gdk-pixbuf/-/blob/2.42.8/meson.build#L9

We might have to investigate if we still should have a different version for `osx` and `arm64`

https://github.com/AnacondaRecipes/gdk-pixbuf-feedstock/blob/cb_2.42.8/recipe/meta.yaml#L29